src/components/MobileNavbar.vue

```diff
@@ -62,3 +69,24 @@ export default {
   }
 }
 </script>
+
+<style scoped>
+  .logo {
+    cursor: pointer;
+  }
+  .item-icon {
+    padding-left:8px;
+    margin-top:11px !important;
```

Comment:
Nigdy, ale to przenigdy nie używaj importantów.

Jak np. masz bibliotekę, która dopisuje do znaczników inline'owe style, których nie nadpiszesz. Wtedy można używać `!important`. Staraj się natomiast ich unikać.

Reply:
Masz rację. Zamiast tego skorzystałem z klas zdefiniowanych przez vuetify.